docs/source/getting_started/using_entitysets.ipynb

```diff
@@ -71,7 +71,19 @@
     "\n",
     "* The ``index`` parameter specifies the column that uniquely identifies rows in the dataframe.\n",
     "* The ``time_index`` parameter tells Featuretools when the data was created.\n",
-    "* The ``logical_types`` parameter indicates that \"product_id\" should be interpreted as a Categorical column, even though it is just an integer in the underlying data."
+    "* The ``logical_types`` parameter indicates that \"product_id\" should be interpreted as a Categorical column, even though it is just an integer in the underlying data.\n",
+    "\n",
+    "You can also use a setter on the ``EntitySet`` object to add dataframes\n",
+    "\n",
+    "```python\n",
+    "es[\"transactions\"] = transactions_df\n",
+    "```\n",
+    "\n",
+    "Note, that this will use the default implementation of `add_dataframe`, notably the following:\n",
+    "\n",
+    "* if the DataFrame does not have [Woodwork](https://woodwork.alteryx.com/) initialized, the first column will be the index column\n",
+    "* if the DataFrame does not have Woodwork initialized, all columns will be inferred by Woodwork.\n",
+    "* control over time index and logical types will need be initialized through Woodwork before adding."
```

Comment:
Suggested rewording:
`if control over the time index column and logical types is needed, Woodwork should be initialized before adding the dataframe`

Reply:
fixed